app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,42 @@
+<% content_for :page_title, "Edit " + resource_name.to_s.humanize %>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email, class: "label" %>
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "input" %>
+  </div>
+
+  <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
+    <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
+  <% end %>
+
+  <div class="field">
+    <%= f.label :password, class: "label" %> <i class="is-size-7">(leave blank if you don't want to change it)</i>
```

Comment:
Maybe move this to a separate view, or toggle it when user requests it, in order to avoid incorrect password changes and  so that we don't have to specify to leave it blank.